pinc/unformat.inc

```diff
@@ -0,0 +1,42 @@
+<?php
+function is_format_round($label)
```

Comment:
We do this "is this round a formatting/proofreading round" in several places (eg: round_project_listing_presort() in `pinc/stages.inc`). We should move this into the Round class in `pinc/RoundDescriptor.inc`, something like:
```
    function is_proofreading_round()
    {
        return startswith($this->id, 'P');
    }

    function is_formatting_round()
    {
        return startswith($this->id, 'F');
    }
```

Reply:
`RoundDescriptor.inc` is generic, but the P/F distinction is pgdp-specific. Putting the code in `stages.inc` would be more appropriate, because that's completely pgdp-specific.